theatre/studio/src/panels/SequenceEditorPanel/RightOverlay/Marker.tsx

```diff
@@ -0,0 +1,83 @@
+import type {Pointer} from '@theatre/dataverse'
+import {useVal} from '@theatre/react'
+import {pointerEventsAutoInNormalMode} from '@theatre/studio/css'
+import getStudio from '@theatre/studio/getStudio'
+import {lockedCursorCssPropName} from '@theatre/studio/uiComponents/PointerEventsHandler'
+import useContextMenu from '@theatre/studio/uiComponents/simpleContextMenu/useContextMenu'
+import useRefAndState from '@theatre/studio/utils/useRefAndState'
+import React from 'react'
+import styled from 'styled-components'
+import {attributeNameThatLocksFramestamp} from '@theatre/studio/panels/SequenceEditorPanel/FrameStampPositionProvider'
+import type {SequenceEditorPanelLayout} from '@theatre/studio/panels/SequenceEditorPanel/layout/layout'
+
+const Mark = styled.div<{left: number}>`
+  position: absolute;
+  height: 20px;
+  width: 8px;
+  background-color: white;
+  z-index: 1;
+  color: black;
+  overflow: hidden;
+  font-size: 10px;
+  left: ${({left}) => left}px;
+
+  ${pointerEventsAutoInNormalMode};
+
+  #pointer-root.draggingPositionInSequenceEditor & {
+    pointer-events: auto;
+    cursor: var(${lockedCursorCssPropName});
+  }
+
+  &:hover {
+    background-color: red;
+  }
+`
+
+const Marker: React.VFC<{
+  layoutP: Pointer<SequenceEditorPanelLayout>
+  marker: {time: number}
```

Comment:
Hmm: A lot of `{time: number}` repeated types, but it doesn't really bother me, since I don't necessarily want to commit to which models are actually shared for these signatures.

Reply:
Noted and discussed briefly.